### DISCUSSION TITLE
Any idea how to fetch my Strapi content to my ecommerce application (bigcommerce)?

### DISCUSSION THREAD

**Discussion by dobradovic** [strapi/strapi#8813] [Legacy Topics]
I am very new to Strapi and my question is how can I import content from Strapi to my bigcommerce theme?

Strapi is headless CMS so it can be implemented with every framework or ecommerce platform, correct me if I am wrong? :)

In my bigcommerce theme I have config.json where I have these fields :

- I doubt that this is not correctly set, because I dont see other way to connect strapi with my bigcommerce theme.

```
"settings": {
"strapi_base_url": "what should I put here?(is it url with my cms admin strapi url?)",
"integration_server_base_url": "And same question, what is integration_server_base_url, where I can find it?",
"google_captcha_key": "/",
"bc_app_client_id": "/",
"google_site_verification": "/",
...
}
```

In short I need to populate a theme with content from Strapi.

Thanks on your time, and every advice would be nice! :)

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
